### ISSUE TITLE
SHA256 problems on Azure

### ISSUE THREAD

**Issue by slochower** [conda-forge/paprika-feedstock#3]
Issue:
Hi @conda-forge/core,

I'm seeing an SHA256 mismatch on my [Azure builds](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=57293). I'm not sure where to start helping debugging this. The only change I made is bumping the version number of my package and I just re-forked the feedstock, so everything should be up to date. Can you point me in the right direction? Thanks!

```
[...]
Success
SHA256 mismatch: 'b423e994467b3004104e437389addb17d7e9b72553d130abff7e0bbf11bdc3aa' != '22a97f6362b6ef9aea9210bd811e2c713dc29158bb61d17af9568a052778cf49'
##[error]Bash exited with code '1'.
```

**Comment by mariusvniekerk** [comment]
So....
GitHub generated generated archives are not eternal. In practice this means that their shad expire every few months. 
If you want a non expirable Sha you have to use a github release or get a tarball from another source

**Comment by isuruf** [comment]
It's not that issue @mariusvniekerk. @slochower you need to update the SHA256 of the archive when you update the version.

**Comment by slochower** [comment]
Ah! My mistake. Updating momentarily! Thanks @mariusvniekerk @isuruf .

**Comment by slochower** [comment]
Okay, great. I'd like to merge the PR. Is there a special place I should mark that is is _okay_ that tests are failing because they depend on stuff outside the conda-forge ecosystem?

**Comment by mariusvniekerk** [comment]
Ideally those things should probably be in the conda forge ecosystem. If they are not there then other users can't easily use this package.

**Comment by slochower** [comment]
Agreed & agreed. But their maintenance is unfortunately out of my control. We are working on it, though ([for example](https://github.com/pandegroup/openmm/issues/2258))!

**Comment by isuruf** [comment]
Can you make those dependencies optional? i.e. package would have less features if those dependencies are not installed, but still work.

**Comment by slochower** [comment]
@isuruf not really, unfortunately. At least, not yet. This package sets up, runs, and analyzes simulations. Without the simulation code, it really wouldn't work at all (we can't even setup the simulations, because we need to know something about how the simulations work). But it looks like `ambertools` now has a working conda-forge [package](https://anaconda.org/conda-forge/ambertools) and hopefully `openmm` will be coming along shortly (held up due to figuring out how to deal with various CUDA installations, whether through `conda` or through NVIDIA).

**Comment by mariusvniekerk** [comment]
One thing you could do if you want to is to patch the conda-forge version of the package so that it warns users that this package needs additional dependencies and gives some degree of instruction on how to install them.

**Comment by slochower** [comment]
@mariusvniekerk interesting idea. How would I do that? Would I need to make separate releases on GitHub just for conda-forge then?

**Comment by jakirkham** [comment]
The `openmm` folks have added their package to conda-forge with GPU support. Figured you'd be happy to know that. 🙂

**Comment by slochower** [comment]
Oh, right, yes of course. I've been following the thread (about whether to include the OpenCL back end...) . Thanks for following up here, much appreciated. I will update this conda recipe soon.
